src/components/interactive-builder/draggable-question.component.tsx

```diff
@@ -90,11 +101,7 @@ const DraggableQuestion: React.FC<DraggableQuestionProps> = ({
           hasIconOnly
           iconDescription={t('editQuestion', 'Edit question')}
           kind="ghost"
-          onClick={() => {
-            if (!isDragging) {
-              handleEditButtonClick(question);
-            }
```

Comment:
Should we factor in `isDragging` when determining whether to launch the modal?

Reply:
Would there be a scenario where the edit button gets clicked while a question is being dragged?

Reply:
Yeah, that makes sense.